www-client/qutebrowser/qutebrowser-1.8.1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{3_5,3_6,3_7} )
+
+inherit distutils-r1 eutils xdg-utils
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/${PN}/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/${PN}/${PN}/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="A keyboard-driven, vim-like browser based on PyQt5 and QtWebEngine"
+HOMEPAGE="https://www.qutebrowser.org/ https://github.com/qutebrowser/qutebrowser"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="scripts test"
+
+COMMON_DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
I doubt this is true, because setuptools should be BDEPEND and not DEPEND. I accept it can go to RDEPEND, but I'll need to compile and check if that is true either.

Reply:
Aggree I will change it to BDEPEND